Assets/Scripts/PauseController.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.UI;
+
+public class PauseController : MonoBehaviour {
+
+    public Transform pauseMenu;
```

Comment:
Instead of storing the Transform (for which you have no use at all), why not storing the GameObject directly?